src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx

```diff
@@ -83,10 +83,12 @@ class DowntimeNotification extends React.Component {
   };
 
   render() {
-    const { appTypeContent } =
-      this.props.customText?.appType || APP_TYPE_DEFAULT;
+    const { formConfig } = this.props;
```

Comment:
Again, just wanted to point out there's no `formConfig` prop in this component.

This component directly takes a  `customText` prop, not a `formConfig` prop that has a `customText` attribute. Refer to the lines deleted to see how it was being used.

So the only change needed here was just removing the curly braces around `appTypeContent` to fix the bug (and optionally change `appTypeContent` to `appType` as you've done).

Reply:
I see what you are referring to. I originally set that up to have the customText be passed in by itself but I was thinking if it would be more beneficial to just pass in the full `formConfig`.

Reply:
@U-DON do you know if the `formConfig` would be a prop [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/monitoring/DowntimeNotification/components/Banner.jsx#L17)?

Reply:
I don't think we can guarantee `formConfig` being passed in as a prop. There are non-form apps that use the downtime components, and it'd be better support custom text for any app.